config/locales/en.yml

```diff
@@ -175,6 +175,10 @@ en:
       ‘Remove from overdue list‘ reason must be one of following:
     list: "Patient not responding, Phone number does not work, Transferred to another public health facility, Moved to a private practitioner, Other reason"
 
+  overdue_patients_return_to_care:
+    reports_card_subtitle: "Overdue hypertension patients that returned to care within 15 days of receiving their first call"
+    numerator: "Patients that returned to care within 15 days of their first call."
+    denominator: "Hypertension patients assigned to %{region_name} with at least one ‘Result of call...’ marked during the month. Deleted, dead and lost to follow-up are excluded."
```

Comment:
Yeah, it had to be a little verbose because there are several key points that we wanna mention here like `hypertension`, `assigned`, `at least one ‘Result of call...’` , `during the month`